proto/zepben/protobuf/cim/iec61970/base/wires/SynchronousMachineKind.proto

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.cim.iec61970.base.wires";
+option csharp_namespace = "Zepben.Protobuf.CIM.IEC61970.Base.Wires";
+package zepben.protobuf.cim.iec61970.base.wires;
+
+/**
+ * Synchronous machine type.
+ */
+message SynchronousMachineKind {
+
+    /**
+     * NOTE: This enum is wrapped in a message due to a c98 naming conflict with SinglePhaseKind.Y
```

Comment:
the naming conflict will be with UNKNOWN

Reply:
done